.claude/scripts/polarion_client.py

```
@@ -0,0 +1,395 @@
+                    "status": 401
+                }
+
+            if response.status_code >= 400:
+                try:
+                    error_data = response.json()
+                    return {
+                        "error": error_data.get("errors", [{}])[0].get("detail", "Unknown error"),
+                        "status": response.status_code,
+                        "response": error_data
+                    }
+                except:
```

Ruff (0.15.17)

[error] Do not use bare `except`

(E722)